src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -201,6 +212,14 @@ await IsPropertyAccessorOrAnOverride(originalSymbol, solution, cancellationToken
                 }
 
                 return false;
+
+                // Local functions
+                static bool IsConstructorForType(ISymbol possibleConstructor, ISymbol possibleType)
+                {
+                    return possibleConstructor.IsConstructor()
+                        && possibleType is INamedTypeSymbol namedType
+                        && Equals(possibleConstructor.ContainingType.ConstructedFrom, namedType.ConstructedFrom);
+                }
```

Comment:
would like @jasonmalinowski 's eyes here as he knows rename-core better than i do. But i don't see anything that concerns me here.

Reply:
Agreed, one of @jasonmalinowski, @dpoeschl or @ryzngard should validate the changes for rename processing.